src/main/java/org/sunbird/workflow/service/impl/TaxonomyServiceImpl.java

```diff
@@ -116,20 +124,20 @@ public void processMessage(WfRequest wfRequest) {
     }
 
     private String constructTermUpdateURI(String term, String category) {
-       String uri = null;
-        if (!StringUtils.isEmpty(term) && !StringUtils.isEmpty(frameworkId) && !StringUtils.isEmpty(category)){
-            UriComponents  uriComponents = UriComponentsBuilder.fromUriString(host + TERM_UPDATE_URI.replace(Constants.ID, term)).
+        String uri = null;
+        if (!StringUtils.isEmpty(term) && !StringUtils.isEmpty(frameworkId) && !StringUtils.isEmpty(category)) {
+            UriComponents uriComponents = UriComponentsBuilder.fromUriString(host + TERM_UPDATE_URI.replace(Constants.ID, term)).
                     queryParam(Constants.FRAMEWORK, frameworkId).queryParam(Constants.CATEGORY, category).build();
-           uri = uriComponents.toString();
+            uri = uriComponents.toString();
         }
         return uri;
     }
 
     private StringBuilder constructPublishFrameworkURI() {
         StringBuilder builder = null;
-        if (!StringUtils.isEmpty(frameworkId)){
+        if (!StringUtils.isEmpty(frameworkId)) {
             builder = new StringBuilder();
-            UriComponents uriComponents = UriComponentsBuilder.fromUriString(host + PUBLISH_FRAMEWORK_URI.replace(Constants.ID,frameworkId)).build();
+            UriComponents uriComponents = UriComponentsBuilder.fromUriString(host + PUBLISH_FRAMEWORK_URI.replace(Constants.ID, frameworkId)).build();
```

Comment:
UriComponentBuilder is unnecessary -- you can directly use the StringBuilder instead.